src/api-service/__app__/onefuzzlib/workers/pools.py

```diff
@@ -227,7 +227,6 @@ def halt(self) -> None:
             delete_queue(self.get_pool_queue(), StorageType.corpus)
             ShrinkQueue(self.pool_id).delete()
             logging.info("pool stopped, deleting: %s", self.name)
-            self.state = PoolState.halt
```

Comment:
Why do we need to stop doing this?

Reply:
Immediately below this, we call `self.delete()`  This is effectively a NOOP.

Reply:
That makes more sense. So this incidental cleanup, unrelated to the larger fix.